docs/index.md

```diff
@@ -6,11 +6,11 @@ tagline: Interested on learning Vue.js? This bootcamp will walk you through a pr
 actionText: Get Started →
 actionLink: /intro/
 features:
-  - title: Why Vue.js?
-    details: Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s
   - title: Why a bootcamp?
-    details: Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s
-  - title: Vue.js Advanced features
-    details: Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s
+    details: Even though you can find multiple ways to learn this technology, a Bootcamp is an easy, practical and fast way to do it. This one is even more special as was built by developers aiming to learn Vue.js, its basic but also advanced topics.
```

Comment:
_its_ should be **it's**

Reply:
No, I can re-state the sentence but in this case is a possessive, I'm referring to Vue.js and its basic and/or advanced topics. Let me change it so that it can be more clear